concurrent/go122.go

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
this should fail our license check in conform... if it doesn't something is broken in conform/kres, and we need to investigate it